paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1279,19 +1281,28 @@ def get_mesos_utilization_error(
         return 0
 
     log.debug(repr(region_pool_utilization_dict))
-    free_pool_resources = region_pool_utilization_dict['free']
-    total_pool_resources = region_pool_utilization_dict['total']
     usage_percs = []
-    for free, total in zip(free_pool_resources, total_pool_resources):
+    for resource in ResourceInfo._fields:
+        free = getattr(region_pool_utilization_dict['free'], resource)
+        total = getattr(region_pool_utilization_dict['total'], resource)
+
         if math.isclose(total, 0):
             continue
+
         current_load = total - free
-        boosted_load = cluster_boost.get_boosted_load(region=region, pool=pool, current_load=current_load)
+
+        # We apply the boost only on the cpu resource.
```

Comment:
I imagine this will end up causing confusion. Our other autoscalers operate on all of the resources. I can easily see us running into a situation where we are out of memory/disk for some reason and someone tries running the cluster boost tool as a quick fix.

Reply:
Keep in mind our main and only use case for now. I'm happy to re-visit the feature when a second use case shows up